build/travis/job2_AppImage/bintray.sh

```diff
@@ -148,12 +148,19 @@ echo "* DESKTOP $DESKTOP"
 #fi
 
 if [ "${APPNAME}" == "MuseScore" ]; then
-  # Get description from desktop file
-  DESCRIPTION=$(bsdtar -f "${FILE}" -O -x ./"${DESKTOP}" | grep -e "^Comment=" | sed s/Comment=//g)
-elif [ "${APPNAME}" == "MuseScoreNightly" ]; then
-  # Use custom description for nightly builds
-  DESCRIPTION="Automated builds of the $BRANCH development branch for $SYSTEM Linux systems. FOR TESTING PURPOSES ONLY!"
+  # Get description from desktop file (source file: build/Linux+BSD/mscore.desktop.in)
+  DESCRIPTION="$(bsdtar -f "${FILE}" -O -x ./"${DESKTOP}" | grep -e "^Comment=" | sed s/Comment=//g)!"
+else
+  # Use custom description for nightly/development builds
+  DESCRIPTION="Automated builds of the $BRANCH development branch. FOR TESTING PURPOSES ONLY!"
 fi
+# Add installation instructions to the description (same for all types of build)
+DESCRIPTION="${APPNAME} Portable AppImages for $SYSTEM Linux systems.
+
+${DESCRIPTION}
+
+Simply download the .AppImage file, give it execute permission, and then run it!
+For more information, see: https://musescore.org/en/handbook/installation-0#linux"
```

Comment:
Not ready to merge until we know the actual URL will be. (See [node 102131](https://musescore.org/en/node/102131).)